hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodiePayloadConfig.java

```diff
@@ -65,9 +65,6 @@ public class HoodiePayloadConfig extends HoodieConfig {
       .withDocumentation("Table column/field name to order records that have the same key, before "
           + "merging and writing to storage.");
 
-  /** @deprecated Use {@link #PAYLOAD_CLASS_NAME} and its methods instead */
-  @Deprecated
-  public static final String DEFAULT_PAYLOAD_CLASS = PAYLOAD_CLASS_NAME.defaultValue();
```

Comment:
Regarding this variable for backwards compatible read (https://github.com/apache/hudi/pull/11943/files#r1805475755), could we hard code it to be `DefaultHoodieRecordPayload.class.getName()`?

Reply:
we did that now